pkg/realserver/configure.go

```diff
@@ -422,6 +606,11 @@ func (r *realserver) checkConfigParity() (bool, error) {
 	for ip, _ := range r.config.Config {
 		vips = append(vips, string(ip))
 	}
+
+	// and, v6 addresses
+	for ip, _ := range r.config.Config6 {
```

Comment:
config6 is _never_ nil?

Reply:
config checked above for nil, provisioner guaranteed, even if empty (famous last words) to make `config.config4` / `config.config6` maps as it has been doing in the past